### PR TITLE
support ADD_PATH mrt subtypes

### PR DESCRIPTION
**Pull request by DrewGregory**
The following RFCs [rfc7911](https://datatracker.ietf.org/doc/html/rfc7911) [rfc8050](https://datatracker.ietf.org/doc/html/rfc8050)support new MRT message types. Thankfully, the message body is in the same format as their non ADD_PATH counterparts. I added them to the corresponding select statement.

### CODE REVIEW

**Review by zakird — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -1,15 +1,13 @@
-module github.com/zmap/zannotate
+module zannotate
```
Comment (ethack) [inline go.mod:1]: Was the `module` line changed intentionally? After this change I'm getting this error when trying to install zannotate.

```
# go install github.com/zmap/zannotate/cmd/zannotate@latest
go install github.com/zmap/zannotate/cmd/zannotate@latest: github.com/zmap/zannotate@none updating to
        github.com/zmap/zannotate@v0.0.0-20210922043249-9e938c460147: parsing go.mod:
        module declares its path as: zannotate
                but was required as: github.com/zmap/zannotate
```

I can successfully install the commit prior to this pull request.
```
# go install github.com/zmap/zannotate/cmd/zannotate@aae45de181c591172b9403d0d8d1984f9f71664e
```

I think the line should still be `module github.com/zmap/zannotate` unless I'm missing something or doing this wrong.

Comment (zakird) [reply go.mod:1]: I agree, this does seem like a regression. @DrewGregory would you be willing to submit a PR that reverts that one line?

Comment (zakird) [reply go.mod:1]: @ethack this should be resolved now that #29 is merged.

Comment (ethack) [reply go.mod:1]: That did it. Thank you!

**Pull request merged**